tests/integration/test_config.py

```diff
@@ -256,3 +257,31 @@ def test_ignored_commands(cluster):
         node.client.execute_command("cmd1", "a", "b")
     with raises(ResponseError, match='NOCLUSTER No Raft Cluster'):
         node.client.execute_command("cmd2", "a", "b")
+
+
+def test_module_commands(cluster):
```

Comment:
```suggestion
def test_module_command_flags(cluster):
    """
    Test commandspec table is updated after module load/unload
    """
     
```
Let's add a small note here about what we are testing